<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAudioPathDlg</class>
 <widget class="QDialog" name="DlgAudioPathDlg">
  <property name="windowTitle">
   <string>Path Editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="ioHLayout">
     <item>
      <widget class="QRadioButton" name="inputRadioButton">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outputRadioButton">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="deviceLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="deviceComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="channelLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Channels</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="channelComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="typeComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="indexLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Index</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="indexSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
